tagpr.go

```diff
@@ -308,25 +308,28 @@ func (tp *tagpr) Run(ctx context.Context) error {
 		return err
 	}
 
-	changelogMd := "CHANGELOG.md"
 	changelog, orig, err := gch.Draft(ctx, nextVer.Tag(), time.Now())
 	if err != nil {
 		return err
 	}
-	if !exists(changelogMd) {
-		logs, _, err := gch.Changelogs(ctx, 20)
-		if err != nil {
+
+	if tp.cfg.changelog == nil || *tp.cfg.changelog {
```

Comment:
@Songmu If tagpr.changelog configuration is not defined, this condition will be evaluated to true, isn't it? So it does not change the current behaviour.

Reply:
Oops. You are right. So here's what you should do.

- Update `cfg.Reload`
- Keep .tagpr unchagend in this repository

Reply:
@Songmu Reverted .tagpr at [81d0d9b](https://github.com/Songmu/tagpr/pull/86/commits/81d0d9b1398f99355ef832bed53dac346efc89ea).